aspnetcore/release-notes/aspnetcore-5.0.md

```diff
@@ -0,0 +1,476 @@
+
+```xml
+<ItemGroup>
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="5.5.1" />
+</ItemGroup>
+```
+
+The template generated code contains code in `Startup.ConfigureServices` that activates OpenAPI document generation:
+
+[!code-csharp[](~/release-notes/sample/StartupSwagger.cs?name=snippet)]
+
+The `Startup.Configure` method adds the Swashbuckle middleware, which enables the:
+
+* Document generation process.
+* Swagger UI page by default in development mode.
+
+The template generated code won't accidentally expose the API's description when publishing to production.
+
+[!code-csharp[](~/release-notes/sample/StartupSwagger.cs?name=snippet2)]
+
+#### Azure API Management Import
+
+When ASP.NET Core API projects enable OpenAPI, the Visual Studio 2019 version 16.8 Preview 3.2 and later publishing automatically offer an additional step in the publishing flow. Developers who use [Azure API Management](xref:tutorials/publish-to-azure-api-management-using-vs) have an opportunity to automatically import the APIs into Azure API Management during the publish flow:
+
+![Azure API Management Import VS publishing](~/release-notes/static/publish-to-apim.png)
+
+### Better launch experience for web API projects
+
+With OpenAPI enabled by default, the app launching experience (F5) for web API developers significantly improves. With .NET, the web API template comes pre-configured to load up the Swagger UI page. The Swagger UI page provides both the documentation added for the published API, and enables testing the APIs with a single click.
+
+![swagger/index.html view](~/release-notes/static/swagger-ui-page-rc1.png)
+
+## Blazor
+
+### `JSObjectReference` replacement
+
+`JSObjectReference` has been made internal, and `IJSObjectReference` takes its place in the public API. For more information, see the [official announcement (aspnet/Announcements #435)](https://github.com/aspnet/Announcements/issues/435) and <xref:blazor/call-javascript-from-dotnet?view=aspnetcore-5.0#blazor-javascript-isolation-and-object-references>.
+
+### CSS isolation
+
+Blazor now supports defining CSS styles that are scoped to a given component. Component specific CSS styles make it easier to reason about the styles in an app and to avoid unintentional side effects from global styles. <!-- UNCOMMENT CROSS-LINK AFTER https://github.com/dotnet/AspNetCore.Docs/pull/19956 MERGES: For more information, see <xref:blazor/components/css-isolation>.-->
+
+### New `InputFile` component
+
+The `InputFile` component allows reading one or more files selected by a user to be upload. For more information, see <xref:blazor/file-uploads>.
+
+### New `InputRadio` and `InputRadioGroup` components
+
+Blazor has built-in `InputRadio` and `InputRadioGroup` components that simplify data binding to radio button groups with integrated validation. For more information, see <xref:blazor/forms-validation>.
```

Comment:
Can we link directly to https://docs.microsoft.com/aspnet/core/blazor/forms-validation?view=aspnetcore-5.0#radio-buttons ?